### PR TITLE
Enable GitHub Actions

### PR DESCRIPTION
**Pull request by kzys**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by kzys** [comment]
Let me remove old Go versions due to #61. I can address the issue later.

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      with:
+        go-version: ${{ matrix.go }}
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Build
+      run: make all
+
+    # GitHub Actions' VMs don't have KVM. Firecracker itself won't work.
```
Comment (xibz) [inline .github/workflows/go.yml:30]: if we are going to use github actions, I would much prefer to host our own github action instance with kvm to run the tests

Comment (kzys) [reply .github/workflows/go.yml:30]: Yeah. GitHub Actions could potentially replace BuildKite altogether. But for now, I'd like to have some here to make sure firectl works on all Go versions.

**Comment by Kern--** [comment]
I think it's fine to have both github actions (for multiple go versions) and buildkite (for integ tests) like we do in firecracker-containerd. We should create an issue to re-enable go 1.11 once we've found a solution for #61

**Review by Kern-- — APPROVED** [review]

**Pull request merged**